suite-native/device/src/screens/DeviceInfoModalScreen.tsx

```diff
@@ -67,12 +98,22 @@ export const DeviceInfoModalScreen = () => {
 
     if (!deviceModel) return null;
 
+    const currentFwVersion = getFirmwareVersion(device);
+
     const handleGoBack = () => {
         navigation.goBack();
     };
 
-    const stringifiedFirmwareVersion = deviceFirmwareVersion?.join('.');
+    const handleAccessoriesClick = () => {
+        Linking.openURL('https://trezor.io/accessories');
```

Comment:
use openLink hook for this please: https://github.com/trezor/trezor-suite/blob/7e0e9a3ee40b3f8b1cf7fb26897885017d753f32/suite-native/link/src/useOpenLink.ts

Reply:
b6eef5a8b